ovh/resource_dbaas_logs_cluster_test.go

```diff
@@ -0,0 +1,61 @@
+package ovh
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func init() {
+	resource.AddTestSweepers("ovh_dbaas_logs_cluster", &resource.Sweeper{
+		Name: "ovh_dbaas_logs_cluster",
+		F:    testSweepDbaasLogsCluster,
+	})
+}
+
+func testSweepDbaasLogsCluster(region string) error {
+	serviceName := os.Getenv("OVH_DBAAS_LOGS_SERVICE_TEST")
+	if serviceName == "" {
+		log.Print("[DEBUG] OVH_DBAAS_LOGS_SERVICE_TEST is not set. No LDP cluster to sweep")
+		return nil
+	}
+
+	// Nothing to sweep as LDP dedicated cluster can't be created/deleted thru API
+
+	return nil
+}
+
+func TestAccDbaasLogsCluster_basic(t *testing.T) {
+	serviceName := os.Getenv("OVH_DBAAS_LOGS_SERVICE_TEST")
+	config := fmt.Sprintf(
+		testAccDbaasLogsClusterConfig,
+		serviceName,
+	)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheckDbaasLogs(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"ovh_dbaas_logs_cluster.ldp", "service_name", serviceName),
+					resource.TestCheckResourceAttrSet(
+						"ovh_dbaas_logs_cluster.ldp", "dedicated_input_pem"),
+					resource.TestCheckResourceAttrSet(
+						"ovh_dbaas_logs_cluster.ldp", "direct_input_pem"),
+				),
+			},
+		},
+	})
+}
+
+const testAccDbaasLogsClusterConfig = `
+resource "ovh_dbaas_logs_cluster" "ldp" {
```

Comment:
You can add threee more attributes in your resource definition, and three more tests in the acceptance test

```
resource "ovh_dbaas_logs_cluster" "ldp" {
  service_name     = "%s"
  archive_allowed_networks = ["10.0.0.0/16"]
  direct_input_allowed_networks  = ["10.0.0.0/16"]
  query_allowed_networks   = ["10.0.0.0/16"]
}
```

and then test if the created `query_allowed_networks` is equals to  `["10.0.0.0/16"]` and the same things for the others attributes :)